docker/sui-network/docker-compose.yaml

```diff
@@ -0,0 +1,176 @@
+version: "3"
+
+services:
+  validator1:
+    networks:
+      sui-network:
+        ipv4_address: 10.0.0.11
+    image: mysten/sui-node:mainnet-v1.19.1
+    container_name: validator1
+    hostname: validator1
+    environment:
+      - RUST_BACKTRACE=1
+      - RUST_LOG=info,sui_core=info,narwhal=info,narwhal-primary::helper=info,jsonrpsee=error
+      - RPC_WORKER_THREAD=12
+    volumes:
+      - ./genesis/files/validator1-8080.yaml:/opt/sui/config/validator.yaml:ro
+      - ./genesis/files/genesis.blob:/opt/sui/config/genesis.blob:ro
+      - /tmp/sui/db1:/opt/sui/db:rw
+    command:
+      [
+        "/usr/local/bin/sui-node",
+        "--config-path",
+        "/opt/sui/config/validator.yaml",
+      ]
+    restart: on-failure
+    logging:
+      driver: "json-file"
+      options:
+        max-file: "10"
+        max-size: "1g"
+  validator2:
+    networks:
+      sui-network:
+        ipv4_address: 10.0.0.12
+    image: mysten/sui-node:mainnet-v1.19.1
+    container_name: validator2
+    hostname: validator2
+    environment:
+      - RUST_BACKTRACE=1
+      - RUST_LOG=info,sui_core=info,narwhal=info,narwhal-primary::helper=info,jsonrpsee=error
+      - RPC_WORKER_THREAD=12
+    volumes:
+      - ./genesis/files/validator2-8080.yaml:/opt/sui/config/validator.yaml:ro
+      - ./genesis/files/genesis.blob:/opt/sui/config/genesis.blob:ro
+      - /tmp/sui/db2:/opt/sui/db:rw
+    command:
+      [
+        "/usr/local/bin/sui-node",
+        "--config-path",
+        "/opt/sui/config/validator.yaml",
+      ]
+    restart: on-failure
+    logging:
+      driver: "json-file"
+      options:
+        max-file: "10"
+        max-size: "1g"
+  validator3:
+    networks:
+      sui-network:
+        ipv4_address: 10.0.0.13
+    image: mysten/sui-node:mainnet-v1.19.1
+    container_name: validator3
+    hostname: validator3
+    environment:
+      - RUST_BACKTRACE=1
+      - RUST_LOG=info,sui_core=info,narwhal=info,narwhal-primary::helper=info,jsonrpsee=error
```

Comment:
i think we will need `sui_core=debug,sui_network=debug,sui_node=debug,narwhal=debug` on all nodes